lib/openstack_query/runners/query_runner.py

```diff
@@ -24,8 +25,12 @@ class QueryRunner(OpenstackWrapperBase):
     Runner classes encapsulate running any openstacksdk commands
     """
 
-    def __init__(self, connection_cls=OpenstackConnection):
+    # Sets the limit for getting values from openstack
+    _LIMIT_FOR_PAGINATION = 1000
+
+    def __init__(self, marker_prop_func: PropFunc, connection_cls=OpenstackConnection):
         OpenstackWrapperBase.__init__(self, connection_cls)
+        self.page_marker_prop_func = marker_prop_func
```

Comment:
This should just use the derived class's implementation rather than passing a function through `__init__`

Reply:
Could you explain what you mean?
we get the prop func from `prop_handler` - I can set that separately in the derived classes I guess - but we'll have some code duplication

Reply:
picking this up in another PR - issue related #100